lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -1,9 +1,25 @@
-import { RenovateConfig } from '../../../../config';
+import { RenovateConfig, RenovateSharedConfig } from '../../../../config';
 import { logger } from '../../../../logger';
 import { clone } from '../../../../util/clone';
 
+function getSemanticCommitsValue(
+  config: RenovateConfig,
+  onboardingConfig: RenovateSharedConfig
+): boolean {
+  if (typeof onboardingConfig.semanticCommits !== 'boolean') {
+    return typeof config.semanticCommits === 'boolean'
+      ? config.semanticCommits
+      : false;
+  }
+  return onboardingConfig.semanticCommits;
+}
+
 export function getOnboardingConfig(config: RenovateConfig): string {
   const onboardingConfig = clone(config.onboardingConfig);
+  onboardingConfig.semanticCommits = getSemanticCommitsValue(
```

Comment:
Let's only set the value if it's `true`. I plan to default it to `false` in the next major release.